src/main/java/uk/gov/hmcts/reform/refunds/config/security/utils/RefundServiceRoles.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.refunds.config.security.utils;
+
+public enum RefundServiceRoles {
+
+    DIVORCE("Divorce", "payments-refund-divorce", "payments-refund-approverr-divorce"),
```

Comment:
remove the extra 'r' in approver role for divorce

Reply:
This class is not used any more so removed.